src/ui/units/wizard/containers/Wizard/SectionVisualization/VisualizationItem/VisualizationItem.tsx

```diff
@@ -333,7 +338,10 @@ class VisualizationItem extends React.Component<Props, State> {
                 <div
                     className={`item-icon ${item.data_type}-icon`}
                     onClick={(event) => {
-                        if (isParameter(item)) {
+                        const isEmptyQaDialog =
+                            this.props.qlChartType && emptyQaDialogTypes.includes(item.data_type);
```

Comment:
You can't rely on that
In some visualizations, date formatting is used, and the table has settings for all fields - for example, a hint